build.gradle

```diff
@@ -368,6 +368,7 @@ project('spring-rabbit-test') {
 
 		compile project(":spring-rabbit")
 		compile ("junit:junit:$junit4Version") {
+			optional(it)
 			exclude group: 'org.hamcrest', module: 'hamcrest-core'
 		}
 		compile "org.hamcrest:hamcrest-all:$hamcrestVersion"
```

Comment:
May we need to make this dependency optional as well?
The same might be applied for the `mockito-core` one, too...

Reply:
Well, perhaps - but the reason for making JUnit4 optional here is so they can use 5 without excluding 4. Let's think about it later.